sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -297,13 +297,13 @@ case class DropNamespace(
 case class DescribeNamespace(
     namespace: LogicalPlan,
     extended: Boolean,
-    override val output: Seq[Attribute] = DescribeNamespace.getOutputAttr) extends Command {
+    override val output: Seq[Attribute] = DescribeNamespace.getOutputAttrs) extends Command {
   override def children: Seq[LogicalPlan] = Seq(namespace)
 
 }
 
 object DescribeNamespace {
-  def getOutputAttr: Seq[Attribute] = Seq(
+  def getOutputAttrs: Seq[Attribute] = Seq(
```

Comment:
When check the code found name not consistence. Just change in this pr or make a followup? @cloud-fan

Reply:
It's very trivial. I'm OK to fix the typo here.